### PR TITLE
docs(sdd): add sdd document

### PR DESCRIPTION
**Pull request by romanmandryk**

### CODE REVIEW

**Review by GoodluckH — CHANGES_REQUESTED** [review]

File: docs/sdd.md [hunk 1]
```diff
@@ -1,3 +1,151 @@
...
+
+Attribute | Description
+--- | ---
+Status | :yellow_circle: Development
+
+This document details the software implementation of `svc-scheduler` (scheduler module).
+
+The scheduler module is responsible for:
+
+- finding feasibility of flight routes (using router library) and estimating flight times
```
[the remark this answers is not in the document]
Comment (romanmandryk) [reply docs/sdd.md]: added link

File: docs/sdd.md [hunk 2]
```diff
@@ -1,3 +1,151 @@
...
+## Location
+
+This document is located under [`svc-scheduler`](https://github.com/Arrow-air/svc-scheduler) repository's [`/docs`](https://github.com/Arrow-air/svc-scheduler/tree/develop/docs) folder.
+
+## Module Attributes
+
+| Attribute       | Applies | Explanation                                                                        |
+|-----------------|---------|------------------------------------------------------------------------------------|
+| Safety Critical | No      | Scheduler is business critical but has no direct impact to the operational safety. |
+| Realtime        | No      |                                                                                    |
```
[the remark this answers is not in the document]
Comment (romanmandryk) [reply docs/sdd.md]: Ok, makes sense. I added the explanation.

**Review by GoodluckH — APPROVED** [review]
LGTM. Perhaps have @ServiceDog take a look at the mermaid.

**Comment by ServiceDog** [comment]
Nice diagrams, thank you!

**Review by ServiceDog — APPROVED** [review]

**Pull request merged**